index.bs

```diff
@@ -3195,6 +3342,7 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
 1. Run [=delete sources for unexpired destination limit=] with |sourcesToDeleteForDestinationLimit| and |source|'s [=attribution source/source time=].
 1. Let |isNoised| be true if |source|'s [=attribution source/randomized response=]
     is not null, otherwise false.
+1. [=Remove or update sources with incompatible attribution scope fields from cache=] with |source|.
```

Comment:
Nitpick -- remove the words, "from cache" ? The use of "from" here doesn't work for both remove and update.